spec/components/schema/supplier.yaml

```diff
@@ -41,7 +41,7 @@ components:
         logo_url:
           type: string
           format: uri
-          description: URL to the logo of the supplier.
+          description: URL to the logo of the supplier. Important, the [format_id] needs to be replaced by a valid format id. You can find the list of all available ids in https://github.com/getyourguide/partner-api-spec/wiki/Image-Formats.
```

Comment:
Do you mind adding it to other places as well? Like in the picture.yaml file?

Reply:
Do you mean the link itself ?

Reply:
Yes, exactly. The format_id is already mentioned in the pictures.yaml with instructions to replace with a proper value from the "wiki". Let's add the link there too. WDYT?